frontend/src/views/AuthPage.vue

```diff
@@ -0,0 +1,184 @@
+<script>
+import authService from '../api/auth';
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import { useUserStore } from '@/stores/user';
+
+export default {
+  name: 'auth-page',
+  data: () => {
+    return { isToggled: false };
+  },
+  setup() {
+    const router = useRouter();
+    const userStore = useUserStore();
+
+    const loginForm = ref({
+      email: '',
+      password: '',
+      error: null,
+    });
+
+    const login = async e => {
+      e.preventDefault();
```

Comment:
I do not believe this is necessary if the `.prevent` modifier is used on the event.